src/components/GenreSelect.js

```diff
@@ -0,0 +1,19 @@
+import {useState} from "react";
+
+export function GenreSelect({genres, selectedGenre, onGenreSelectCb}) {
+    const [selectedGenreState, setSelectedGenreState] = useState(selectedGenre);
```

Comment:
`selectedGenre` prop should be used for identifying a genre that is currently selected, not for initialization of an internal state. You don't need `selectedGenreState` at all.